docs/md/flows.md

```diff
@@ -1034,7 +1034,7 @@ will be created and it's the responsibility of the caller to act on that by init
                 },
                 "join_date": {
                     "text": "2017-12-02",
-                    "datetime": "2017-12-02T00:00:00-02:00"
+                    "datetime": "2017-12-02T00:00:00.000000-02:00"
```

Comment:
used to be two ways of JSONifying XValues since a complex thing like @contact could be jsonified for inclusion in the session or jsonified in an expression. Now there's only primitives and they only have one way of JSONifying.